client/post-editor/editor-discussion/test/index.jsx

```diff
@@ -48,7 +48,7 @@ describe( 'EditorDiscussion', function() {
 			recordEvent: noop,
 			recordStat: noop
 		} );
-		EditorDiscussion = require( '../' );
+		EditorDiscussion = require( '../' ).WrappedComponent;
```

Comment:
Why are we importing the wrapped component? If the connected component is not ready to use, should we not be changing it at all here?